packages/paste-website/src/pages/patterns/filter/index.mdx

```diff
@@ -0,0 +1,698 @@
+
+  <ResponsiveImage src={UsageImage} alt="The filter category label should be the same as the form field label" />
+
+  - Check our [Form documentation](/components/form#selection-methods) to choose the best Selection Method for your filter use case.
+- Determine users' intent and their relationship with the product and dataset.
+  - Categorize filters into the most relevant and commonly used filters versus the more advanced or less used ones.
+  - Determine how many different types of users will access the feature and how much the filtering criteria will vary from use case to use case. If necessary, consider allowing users to [add custom filters](https://docs.google.com/document/d/1QcSeyk64EfHFnwVg6onVF-E-IYR1VZ91anZMBHlsNjI/edit#heading=h.h0q1xjjeczp) to provide a more tailored experience.
+  - Identify any filter values that naturally lead to additional refinement options, and consider implementing [conditional filters](https://docs.google.com/document/d/1QcSeyk64EfHFnwVg6onVF-E-IYR1VZ91anZMBHlsNjI/edit#heading=h.ml9y8ac3naoj).
+- Work with your engineer to determine any technical constraints.
+  - Ensure you have a good understanding of how the data is set up, including which filtering behaviors and combinations are not possible.
+  - Evaluate carefully the best filtering behavior for [fetching results](https://docs.google.com/document/d/1QcSeyk64EfHFnwVg6onVF-E-IYR1VZ91anZMBHlsNjI/edit#heading=h.13p2794htv04). Keep in mind user intent, but also loading times and the size of the dataset.
+  - Ensure that all filter options correspond to data present in the results. Avoid including filters for data that are not visible or accessible within the dataset.
+  - Evaluate the expected growth of your filtering criteria and ensure it can manage larger datasets without notable performance decline.
+
+
+### Accessibility
+
+When using the filter pattern, ensure that users:
+- Can easily identify and understand labels for each selection method, filter value, or category.
+- Are notified of state changes. For example, when a button changes from the disabled state to the default state, or when the dataset enters a loading state.
+- Can complete all actions with the keyboard.
+
+<Callout variant="neutral" marginY="space70">
+  <CalloutText>
+    Check accessibility considerations for each component part of the filter feature, for example if you’re using Time Picker check its accessibility guidelines and make sure to follow.
+  </CalloutText>
+</Callout>
+
+## Variations
+
+### Filter bar
+
+A filter bar is the default way of displaying filters
+
+Use it when all filters are relevant and necessary to the search. It can also be used in combination with a [More filters Side Panel](https://docs.google.com/document/d/1QcSeyk64EfHFnwVg6onVF-E-IYR1VZ91anZMBHlsNjI/edit#heading=h.2te8zormvjon) when there are secondary filter criteria.
```

Comment:
```suggestion
A filter bar is the default way of displaying filters.

Use it when all filters are relevant and necessary to the search. It can also be used in combination with a [More filters Side Panel](#more-filters-side-panel) when there are secondary filter criteria.
```